psi-ms.obo

```diff
@@ -23328,45 +23398,57 @@ def: "The definition of the outlier criteria applied." [PSI:MS]
 is_a: MS:4000141 ! outlier threshold criterion
 
 [Term]
 id: MS:4000144
 name: Tukey's fence low outliers
-def: "Defines low outliers with Tukey's fence as <(Q1-x*IQR) for low outliers, where x is defined by the term's value. The default is x=1.5" [PSI:MS]
+def: "Defines low outliers with Tukey's fence as <(Q1-x*IQR) for low outliers, where x is defined by the term's value. The default is x=1.5." [PSI:MS]
 is_a: MS:4000141 ! outlier threshold criterion
 
 [Term]
 id: MS:4000145
 name: Z-score threshold
-def: "Defines outliers with a Z-score threshold as <(-x) for low outliers and >(+x) for high outliers, where x is defined by the term's value. The default is x=3" [PSI:MS]
+def: "Defines outliers with a Z-score threshold as <(-x) for low outliers and >(+x) for high outliers, where x is defined by the term's value. The default is x=3." [PSI:MS]
 is_a: MS:4000141 ! outlier threshold criterion
 
 [Term]
 id: MS:4000146
 name: Z-score threshold high outliers
-def: "Defines outliers with a Z-score threshold as <(-x) for low outliers and >(+x) for high outliers, where x is defined by the term's value. The default is x=3" [PSI:MS]
+def: "Defines outliers with a Z-score threshold as <(-x) for low outliers and >(+x) for high outliers, where x is defined by the term's value. The default is x=3." [PSI:MS]
 is_a: MS:4000141 ! outlier threshold criterion
 
 [Term]
 id: MS:4000147
 name: Z-score threshold low outliers
-def: "Defines outliers with a Z-score threshold as <(-x) for low outliers and >(+x) for high outliers, where x is defined by the term's value. The default is x=3" [PSI:MS]
+def: "Defines outliers with a Z-score threshold as <(-x) for low outliers and >(+x) for high outliers, where x is defined by the term's value. The default is x=3." [PSI:MS]
 is_a: MS:4000141 ! outlier threshold criterion
 
 [Term]
 id: MS:4000148
 name: algorithmical threshold
-def: "Defines outliers algorithmically, where a single value threshold might not be applicable or p.r.n. multivariate decision making is applied. The value of the term should name the algorithmical method used" [PSI:MS]
+def: "Defines outliers algorithmically, where a single value threshold might not be applicable or p.r.n. multivariate decision making is applied. The value of the term should name the algorithmical method used." [PSI:MS]
 is_a: MS:4000141 ! outlier threshold criterion
 
+[Term]
+id: MS:4000149
+name: iRT calibration formula
+def: "A polynomial formula to calibrate retention time based on iRT reference peptides. The order of the values corresponds to polynomial terms. I.e. a linear equation is represented by a two-tuple consisting of (slope, intercept). More general, the position in the n_tuple indicates the power of `x`: position `n → x^0`, position `n - 1 → x^1`, position `n - 2 → x^2`, etc." [PSI:MS]
+relationship: has_value_type MS:4000004 ! The allowed value-type for this CV term
```

Comment:
Replace line by:
```
is_a: MS:4000004 ! n-tuple
relationship: has_metric_category MS:4000008 ! ID based metric
relationship: has_metric_category MS:4000016 ! retention time metric
relationship: has_value_type xsd:float ! The allowed value-type for this CV term
```

Reply:
@mobiusklein I think there's still a difference between the `is_a` for single value or tuple and the `relationship: has_value_type` for floats that's not covered for these two terms, right?

Reply:
The suggested lines look right to me. `is_a` tuple and the tuple values `has_value_types` float.